### PR TITLE
email notif

### PR DESCRIPTION
**Pull request by elimelt**

### CODE REVIEW

File: server/interview/views.py [hunk 1]
```diff
@@ -243,25 +309,29 @@ def post(self, request):
...
+#         logger.debug("POST request received for NotifyInterview")
+#         # TODO: Implement the notification service
+#         logger.info("Notification sent (placeholder)")
+#         return Response({"detail": "Notification sent."})
 
 
 class InterviewQuestions(APIView):
     # TODO: Change permission class to isAdminUser
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsAdminUser]
```
Comment (ShawnCollinge) [inline server/interview/views.py:334]: This is the wrong permission, needs to be `IsAdmin`. This is the import: `from custom_auth.permissions import IsAdmin`

Comment (elimelt) [reply server/interview/views.py]: Just checking, is the `IsAdminUser` in `questions/views.py` intentional?

**Review by ShawnCollinge — APPROVED** [review]
LGTM

File: server/interview/views.py (hunk 1 above)
Comment (ShawnCollinge) [reply server/interview/views.py]: No, that is changed in my PR I'm waiting for you to review :P

**Pull request merged**